### PR TITLE
fix the use of initial learning rate in the OneCycleLR example

### LINKED ISSUE #127649: OneCycleLR Example
---
name: "OneCycleLR Documentation"
about: "Clarify the correct use of initial learning rate in the OneCycleLR example"

---

## OneCycleLR Documentation

https://github.com/pytorch/pytorch/blob/6bfc6e08759cf1fd7cf89916124285bf131b7168/torch/optim/lr_scheduler.py#L1901-L1902

### Description of Issue

In the above example for `torch.optim.lr_scheduler.OneCycleLR`, the optimizer is initialized with `lr=0.1` while `max_lr` for the scheduler is set to `0.01`. This means that the initial learning rate in the optimizer is set higher than the `max_lr` specified in the scheduler.

This is not consistent with the intended use of `OneCycleLR`, which caused me some confusion as I was learning about how to use the `OneCycleLR` scheduler. Specifically, the initial learning rate should start low and then increase to `max_lr` during the warm-up phase.

### Current Example in Documentation
```python
data_loader = torch.utils.data.DataLoader(...)
optimizer = torch.optim.SGD(model.parameters(), lr=0.1, momentum=0.9)
scheduler = torch.optim.lr_scheduler.OneCycleLR(optimizer, max_lr=0.01, steps_per_epoch=len(data_loader), epochs=10)
for epoch in range(10):
    for batch in data_loader:
        train_batch(...)
        optimizer.step()
        scheduler.step()
```

### Suggested Correction
The initial learning rate of the optimizer should be set to a lower value, typically lower than `max_lr`. Here is the corrected example:

```python
data_loader = torch.utils.data.DataLoader(...)
optimizer = torch.optim.SGD(model.parameters(), lr=1e-4, momentum=0.9)
scheduler = torch.optim.lr_scheduler.OneCycleLR(optimizer, max_lr=0.01, steps_per_epoch=len(data_loader), epochs=10)
for epoch in range(10):
    for batch in data_loader:
        train_batch(...)
        optimizer.step()
        scheduler.step()
```

### Rationale
By setting the initial learning rate to a low value (e.g., `1e-4`), the scheduler can properly adjust the learning rate from this low starting point up to the specified `max_lr` (e.g., `0.01`) following the OneCycle learning rate policy.

### Reference
The issue is related to the documentation found in the following section:
[torch.optim.lr_scheduler.OneCycleLR Documentation](https://github.com/pytorch/pytorch/blob/6bfc6e08759cf1fd7cf89916124285bf131b7168/torch/optim/lr_scheduler.py#L1901-L1902)

Please update the example to reflect the proper usage of the `OneCycleLR` scheduler.

Thank you.

cc @svekars @brycebortree @vincentqb @jbschlosser @albanD @janeyx99 @crcrpar

### PR DESCRIPTION
**Pull request by tianyeeT**
Fixes #127649

### CODE REVIEW

**Review by janeyx99 — APPROVED** [review]
looks fine!
